.azure/test-linux.yml

```diff
@@ -84,27 +88,27 @@ jobs:
           env:
             SETUPTOOLS_ENABLE_FEATURES: "legacy-editable"
 
+      - ${{ if eq(parameters.installOptionals, true) }}:
+        - bash: |
+            set -e
+            source test-job/bin/activate
+            python -m pip install -r requirements-optional.txt -c constraints.txt
+          displayName: "Install optional packages"
+
+        - bash: |
+            set -e
+            sudo apt-get update
+            sudo apt-get install -y graphviz
+            pip check
```

Comment:
Nit, the `pip check` should go in the above section

Reply:
Done in 67682df76.